Mapping Tools/Classes/SystemTools/QuickRun/QuickRunEnum.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel;
+
+namespace Mapping_Tools.Classes.SystemTools {
+    public enum SingleQuickRunEnum {
+        [Description("<Current Tool>")]
+        Current = 0,
+        [Description("Map Cleaner")]
+        Cleaner = 1,
+        [Description("Slider Completionator")]
+        Completionator = 2,
+    }
+
+    public enum MultipleQuickRunEnum {
+        [Description("<Current Tool>")]
+        Current = 0,
+        [Description("Map Cleaner")]
+        Cleaner = 1,
+        [Description("Slider Completionator")]
+        Completionator = 2,
+        [Description("Slider Merger")]
+        Merger = 3,
+    }
+}
```

Comment:
I would prefer if you actually made 2 new interfaces that inherit IQuickRun; IQuickRunSingle and IQuickRunMultiple. Any tool implementing those interfaces will work just like IQuickRun but also automatically be added to this list of options for SmartQuickRun. You can use Reflection to get all types that inherit an interface (an example is in SnappingToolsVm).

Reply:
That's better because you would just give a tool an interface instead of having to remember to add it to this enum and switch case.